lib/systemd/system/openfortivpn@.service.in

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=OpenFortiVPN for %I
+After=network-online.target
+
+[Service]
+Type=simple
+PrivateTmp=true
+ExecStart=/usr/bin/openfortivpn -c @SYSCONFDIR@/openfortivpn/%I.conf
```

Comment:
`/usr/bin/openfortivpn` is hard coded here. I believe this should also be something like `@PREFIX@/bin/openfortivpn` or so

Reply:
Fixed now.

Reply:
I have added a @BINDIR@ in the template file.